packages/editor-ui/src/views/NodeView.vue

```diff
@@ -1607,7 +1607,10 @@ export default mixins(
 				await this.$store.dispatch('workflows/setNewWorkflowName');
 				this.$store.commit('setStateDirty', false);
 
-				await this.addNodes([DEFAULT_START_NODE]);
+				// Set default values for the new default node
```

Comment:
comment here is not needed. does not add anything

Reply:
They are removed